packages/hoppscotch-app/components/app/Footer.vue

```diff
@@ -155,6 +155,12 @@
           :title="t('app.shortcuts')"
           @click.native="showShortcuts = true"
         />
+        <!-- <ButtonSecondary
+          v-tippy="{ theme: 'tooltip' }"
+          svg="zap"
+          :title="t('app.shortcuts')"
+          @click.native="showShortcuts = true"
+        />-->
```

Comment:
What is this? I guess this file should not be in this PR. Must've mistakenly added to the commits, kindly revert the changes in this file.